contracts/gov/src/contract.rs

```diff
@@ -503,7 +503,12 @@ pub fn execute_poll_messages(
     poll_store(deps.storage).save(&poll_id.to_be_bytes(), &a_poll)?;
 
     let mut messages: Vec<CosmosMsg> = vec![];
+
     if let Some(all_msgs) = a_poll.execute_data {
+        if all_msgs.is_empty() {
```

Comment:
We have text proposals that do not include execute messages. with this new changes we expire them instead of putting them in execution status. In this case, do we need to add this new change?

Reply:
I agree. I will close this PR and the associated issue for this reason. Good catch Mo :)